### PR TITLE
C++: Remove internal `microsoft` tags from queries

### PR DESCRIPTION
**Pull request by MathiasVP**
This PR removes the internal `microsoft` tags on the remaining seven queries. Now, `grep`'ing for `"external/microsoft"` gives no more results in `.ql` files.

### CODE REVIEW

**Review by jketema — COMMENTED** [review]
LGTM. I assume you'll notify the MS contact before we merge this?

**Comment by MathiasVP** [comment]
> LGTM. I assume you'll notify the MS contact before we merge this?

Yep. I'm writing them an email now.

**Review by jketema — APPROVED** [review]
Got the ok from MS.

**Pull request merged**
